compose/ui/ui/src/commonMain/kotlin/androidx/compose/ui/node/NodeCoordinator.kt

```diff
@@ -470,9 +470,9 @@ internal abstract class NodeCoordinator(
     // implementation of draw block passed to the OwnedLayer
     @Suppress("LiftReturnOrAssignment")
     private val drawBlock: (Canvas, GraphicsLayer?) -> Unit = { canvas, parentLayer ->
-            if (layoutNode.isPlaced) {
-                snapshotObserver.observeReads(this, onCommitAffectingLayer) {
-                    drawContainedDrawModifiers(canvas, parentLayer)
+        if (layoutNode.isPlaced) {
```

Comment:
Let's revert `commonMain` formatting, even if it is incorrect. It should be fixed in AOSP, not in the fork

Reply:
I believe it's caused by merge conflict and AOSP contains correct formatting. But since it's not in the direct scope - ok, will do